communication/contributor-comms/role-handbooks/blog-coordinator.md

```diff
@@ -31,18 +60,68 @@ Those in this role should ideally:
+   team.
 8. Follow up submitted PRs and help with the approval process.
 
 ### Associates Overview (Shadow)
 
-Team members who which to become Blogging Leads should express their desire to the current lead; leads should work with shadows to understand their capacity to take on the needs of the lead role and work with the shadow on a plan to take on responsibility, including:
-
-* Become a reviewer of new submitted content.
-* Open the PRs and follow them through.
-* Provide assistance to new contributors that are asking how to submit content.
+#### Description and eligibility
+
+Anyone that has interested in helping the blogging team, and that
+finds the general expectations and requirements acceptable, can start
+working in the team and have their contributions to the project
+recognised. Additionally, some will feel that they want to make a step
+towards becoming Blogging Leads themselves.
+
+Becoming a shadow is something that should come as a result of
+existing involvement in the team, and not as a starting point; to be
+eligible to shadow for this role you will need to:
+
+* Be a regular member of the Comms teams, by following the
+  activities and the regular meetings.
+* Have demonstrated knowledge of the blogging process and guidelines,
+  namely by having been actively involved in the publishing process of
+  articles.
+
+Team members who want to become Blogging Leads _should express their
+desire to the current lead_; leads should work with shadows to
+understand their capacity to take on the needs of the lead role and
+work with the shadow on a plan to take on the necessary
+responsibility.
+
+#### Expected contributions
+
+Some of the identified requirements for blogging shadows are:
+
+* **Become a reviewer of new submitted content**, by leading communication
+  with content creators and guiding them through the publishing
+  process.
+* **Open the PRs and follow them through**, helping content creators
+  in the more processual aspects of publishing, keeping them informed
+  of the necessary steps and following the process untul completion.
+* **Provide assistance to new contributors that are asking how to
+  submit content**, by monitoring the Slack channel, Github, and other
+  sources.
+* **Make status updates in the SIG Contribex Comms meetings**, keeping
+  the status of the relevant projects up-to-date (e.g., in the project
+  board) and having a broad understanding of the different ongoing tasks.
+
+This requirements are aligned with the general expectations described
+above, but in the context of shadowing they follow a more structured
+approach. The acting Blogging Lead will discuss the steps that need to
+be taken to make the shadowing programme helpful and achieve the desired result.
+
+Changes in roles are done by the SIG Contribex Comms subproject team
+based on different inputs and needs, but having shadows that are ready
+to assume a leading position is a fundamental aspect for allowing it.
```

Comment:
for allowing it to grow? Seems like you're missing something at the end of this sentence.